static/js/stream_edit.js

```diff
@@ -552,6 +552,10 @@ exports.initialize = function () {
 
         function invite_failure(xhr) {
             const error = JSON.parse(xhr.responseText);
+            // The message has beem changed to "Unable to find user" but not in the API
+            if (error.msg.slice(0, 51) === "User not authorized to execute queries on behalf of") {
+                error.msg = "Unable to find user " + error.msg.slice(51);
```

Comment:
This doesn't look correct.  This failure response is generated via `zerver/views/streams.py`, and [this](https://github.com/gnprice/zulip/commit/a0e53d25aba13a3723566ccd3ce99ea8c751dbb8) commit gives some context as to the background.  So it seems we simply don't want to show this particular error message in this case.  Which is as simple as:
```js
stream_subscription_info_elem.text(i18n.t("Unable to find user XXX"));
```
on a failed invite, just grab whatever user was being searched for to replace `XXX`.